docs/mailbox.md

```diff
@@ -0,0 +1,10 @@
+## Message to mailbox
+### The TruckAgent informs the mailbox about the delivered order
+#### String: "OrderID"
+<br>
```

Comment:
A couple of things that are missing that I can think of is the number of boxes and which bakery sent it. Remember a single client may be receiving orders from different bakeries.

Reply:
This has been fixed with the new commit